pkg/controller/disruption/disruption_test.go

```diff
@@ -158,6 +159,7 @@ func newFakeDisruptionController() (*disruptionController, *pdbStates) {
 	fakeDiscovery := &discoveryfake.FakeDiscovery{
 		Fake: &core.Fake{},
 	}
+	fakeClock := clocktesting.NewFakeClock(time.Now())
```

Comment:
Should something maybe advance the fake clock ever? You can continue to use a real clock if that makes things easier?